docs/api/public_php_api.md

```diff
@@ -1,12 +1,12 @@
 # eZ Platform Public PHP API
 
-The PHP API is also commonly referred to as the "*Public API*". Currently it exposes a Repository which allows you to create, read, update, manage and delete all objects available in eZ Platform, first and foremost content, but also related objects like Sections, Locations, Content Types, Content Type groups, languages and so on.
+The PHP API is also commonly referred to as the **public API**. Currently it exposes a Repository which allows you to create, read, update, manage and delete all objects available in eZ Platform. First and foremost content, but also related objects like Sections, Locations, Content Types, Content Type groups, languages and so on.
 
 ### eZ Platform API Repository
 
-This entity is the entry point to everything you will do with the Public API.
+This entity is the entry point to everything you do with the public API.
 
-It will allow you to create, retrieve, update and delete all the eZ Platform objects, as well as Content Types, Sections, Content states. It is always obtained through the service container.
+It allows you to create, retrieve, update and delete all the eZ Platform objects, as well as Content Types, Sections, Content states. It is always obtained through the service container.
```

Comment:
Not specifically something to fix here, but in general I'd tend towards "enable" instead of "allow", unless referring to permissions specifically